### ISSUE TITLE
Match icons with themes

### ISSUE THREAD

**Issue by mradamcox** [healthyregions/SDOHPlace#287]
We need to match the new icons @shubhamk008 has selected with the 12 themes that will be implemented as a controlled vocabulary for that field.

**Comment by pengyin-shan** [comment]
The themes will be added as a new attribute to the Solr object, right?

**Comment by mradamcox** [comment]
No, there is an existing field `subject` which will hold what we are calling "themes" here (this is a little confusing because there is a different field `theme`, but that one has a pre-defined controlled vocabulary, so we are using `subject` which allows us to create our own controlled vocabulary).

Thing is, we still need to fix the existing metadata records so that the `subject` field properly reflects the options listed above.

**Comment by mradamcox** [comment]
I think the best way to implement this on the frontend for now would just be a lookup table that matches these values with icons, and ignores any `subject` values that don't match the pre-made lookup. That way we aren't dependent on fixing all the metadata, but once it does it fixed the system will display it properly.

**Comment by pengyin-shan** [comment]
Currently, I have an IconMatch method to generate icons based on certain strings:
<img width="575" alt="Screenshot 2024-09-05 at 11 28 20 AM" src="https://github.com/user-attachments/assets/50bbac79-de7d-445c-a5b6-177a7ec950f3">

Once we are done with the subject values in Solr, these values can be added to this method and matched with the icons Shubhma provided.

**Comment by mradamcox** [comment]
Ok, great, that will work for sure.
